scala/private/phases/phase_scala_provider.bzl

```diff
@@ -0,0 +1,45 @@
+#
+# PHASE: scala provider
+#
+# DOCUMENT THIS
+#
+load(
+    "@io_bazel_rules_scala//scala:providers.bzl",
+    "create_scala_provider",
+)
+
+def phase_library_scala_provider(ctx, p):
+    args = struct(
+        rjars = depset(
+            transitive = [p.compile.rjars, p.collect_exports_jars.transitive_runtime_jars],
+        ),
+        compile_jars = depset(
+            p.compile.ijars,
+            transitive = [p.collect_exports_jars.compile_jars],
+        ),
+    )
+    return phase_common_scala_provider(ctx, p, args)
+
+def phase_common_scala_provider(ctx, p, _args = struct()):
+    return _phase_scala_provider(
+        ctx,
+        p,
+        _args.rjars if hasattr(_args, "rjars") else p.compile.rjars,
```

Comment:
did we lose this https://github.com/bazelbuild/rules_scala/pull/865/files#diff-d68dd15f790bd2a815ab04d3d0a320d8L843?

Reply:
That was handled in `phase_compile.bzl`
https://github.com/bazelbuild/rules_scala/pull/865/files#diff-78d7aa1b58ceeb6b93b4a167504e05c4R140